schema/atop_xslt.xspec

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<x:description xmlns:x="http://www.jenitennison.com/xslt/xspec" schematron="atop_xslt.sch">
+    <x:scenario label="Validating global names">
```

Comment:
I do not write XSpec, yet, and have barely even read any before now. (Mostly snippets on the screen at conference talks!) That said, shouldn’t we have a scenario for a name with the proper prefix and namespace URI whose local-name does not match the desired regex? E.g. something like
~~~xml
    <x:scenario label="A global variable with an inappropriate local name">
      <x:context>
        <xsl:stylesheet version="3.0" 
          xmlns:atop="http://www.tei-c.org/ns/atop"
          xmlns:xsl="http://www.w3.org/1999/XSL/Transform">
          <xsl:variable name="atop:eLbairaVlabolGv" select="true()"/>
        </xsl:stylesheet>
      </x:context>
      <x:expect-assert id="assert-global-name-prefix"/>
      <x:expect-assert id="assert-global-name-uri"/>
    </x:scenario>
~~~

Reply:
Right: [](https://github.com/TEIC/atop/pull/5/commits/612fa9d9ad59eb51dbc18a7f0351ba6d59ecb1ce)[612fa9d](https://github.com/TEIC/atop/pull/5/commits/612fa9d9ad59eb51dbc18a7f0351ba6d59ecb1ce)

Reply:
My first XSpec code. 😁